portability-spi-transfer/src/main/java/org/datatransferproject/spi/transfer/types/TempPhotosData.java

```diff
@@ -19,19 +19,17 @@
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.annotation.JsonTypeName;
-import java.util.Collection;
-import java.util.HashMap;
-import java.util.LinkedHashSet;
-import java.util.Map;
-import java.util.UUID;
 import org.datatransferproject.types.common.models.DataModel;
 import org.datatransferproject.types.common.models.photos.PhotoAlbum;
 
+import java.io.Serializable;
+import java.util.*;
+
 /*
  * TempPhotosData used to store album and photos information before they are ready to be uploaded.
  */
 @JsonTypeName("org.dataportability:TempPhotosData")
-public class TempPhotosData extends DataModel {
+public class TempPhotosData extends DataModel implements Serializable {
```

Comment:
would it be simpler to make DataModel implement Serializable? Then I think the subclasses get it for free?

Reply:
I made PortableType implement serializable so that should cover it.